system/services/assetManager/NativeImageService.cfc

```diff
@@ -193,7 +194,24 @@ component displayname="Native Image Manipulation Service" {
 			}
 		}
 
-		pdf attributeCollection=pdfAttributes;
+		if ( SERVER.lucee.version.left(1) gt 4 ) {
+			tmpFilePDF  = GetTempDirectory() & imagePrefix & "_page_" & arguments.page & ".pdf";
```

Comment:
I think `tmpFilePDF` and `tmpFileJPG` are missing `var` scope. Will fix after merge.